client/state/plugins/premium/test/reducer.js

```diff
@@ -0,0 +1,453 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PLUGIN_SETUP_FETCH_INSTRUCTIONS,
+	PLUGIN_SETUP_RECEIVE_INSTRUCTIONS,
+	PLUGIN_SETUP_INSTALL,
+	PLUGIN_SETUP_ACTIVATE,
+	PLUGIN_SETUP_CONFIGURE,
+	PLUGIN_SETUP_FINISH,
+	PLUGIN_SETUP_ERROR
+} from 'state/action-types';
+
+import { isRequesting, plugins } from '../reducer';
+
+const initSite = [
```

Comment:
;) I personally like these inline since it reduces context switching. It's fine in this case. If these test fixtures get too large we can maybe consider adding in some helper test factories.